scripts/build_component_container.sh

```diff
@@ -23,14 +24,22 @@ if make -C "${executable_src_path}" --no-print-directory --dry-run container >/d
 then
     make -C "${executable_src_path}" --no-print-directory --quiet container CONTAINER_REPO="${CONTAINER_REPO}" DRACON_VERSION="${DRACON_VERSION}"
 else
-    dockerfile_template="
-        FROM ${BASE_IMAGE:-scratch}                     \n
-        COPY ${executable_path} /app/${executable_path} \n
-        ENTRYPOINT ["/app/${executable_path}"]          \n
-    "
+    dockerfile_template=""
```

Comment:
I have a lot of reservations about these changes. this looks like it can be simplified to just:
1. I don't see why there should be 2 Dockerfiles, we can just have one based on alpine and if the user wants a different base to wrap all the executables they can supply a different `BASE_IMAGE_PATH`. Which would simplify this to:
```bash
BASE_IMAGE_PATH=${BASE_IMAGE_PATH:-./containers/Dockerfile.base}
docker build \
        --build-arg executable_path=${executable_path}\
        -t "${CONTAINER_REPO}/${executable_src_path}:${DRACON_VERSION}" \
        $([ "${SOURCE_CODE_REPO}" != "" ] && echo "--label=org.opencontainers.image.source=${SOURCE_CODE_REPO}" ) \
        -f "${BASE_IMAGE_PATH}" ./bin
```

Also, it's customary for shell environment variables to be uppercase, so I think the `executable_path` in the Dockerfile should be uppercase

This should simplify the whole PR quite a bit